src/isolate.rs

```diff
@@ -282,23 +289,77 @@ impl Isolate {
     }
   }
 
+  /// Runs the default MicrotaskQueue until it gets empty.
+  /// Any exceptions thrown by microtask callbacks are swallowed.
+  pub fn run_microtasks(&mut self) {
+    unsafe { v8__Isolate__RunMicrotasks(self) }
+  }
+
+  /// Enqueues the callback to the default MicrotaskQueue
+  pub fn enqueue_microtask(&mut self, microtask: Local<Function>) {
+    unsafe { v8__Isolate__EnqueueMicrotask(self, microtask) }
+  }
+
+  /// Disposes the isolate.  The isolate must not be entered by any
+  /// thread to be disposable.
+  unsafe fn dispose(&mut self) {
+    IsolateHandle::dispose(self);
+    v8__Isolate__Dispose(self)
+  }
+}
+
+// TODO(ry) Use AtomicPtr? Use Cell? Only one thread ever modifies it.
```

Comment:
> What's the "it" that only one thread ever modifies?

Reply:
it = IsolateHandle

Reply:
The IsolateHandle never gets modified by any thread. What gets modified?

Reply:
take() modifies it in IsolateHandle::dispose